Library/Homebrew/test/formula_spec.rb

```diff
@@ -699,6 +699,38 @@
     end
   end
 
+  specify "#service" do
+    f = formula do
+      url "https://brew.sh/test-1.0.tbz"
+      service do
+        run ["/usr/local/bin/beanstalkd"]
```

Comment:
```suggestion
        run [bin/"beanstalkd"]
```

because that's what we'll get people to do in formulae (and it'd be good to handle `Pathname` here too)

Reply:
How would I need to handle `Pathname`?